packages/realm-web/src/User.ts

```diff
@@ -331,7 +331,10 @@ export class User<
   }
 
   /** @inheritdoc */
-  public mongoClient(serviceName: string): Realm.Services.MongoDB {
+  public mongoClient(serviceName: string, options?: { extraHeaders?: { string: string }}): Realm.Services.MongoDB {
```

Comment:
There should be a `headers` type that can be used by the three introduced changes. 
`type headers = { [key: string]: string }`

Reply:
nice, will do.